pkg/controllers/networkpolicy/networkpolicy_controller.go

```diff
@@ -167,7 +167,7 @@ func (r *NetworkPolicyReconciler) CollectGarbage(ctx context.Context) {
 	for elem := range diffSet {
 		log.V(1).Info("GC collected NetworkPolicy", "ID", elem)
 		metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerDeleteTotal, MetricResType)
-		err = r.Service.DeleteSecurityPolicy(types.UID(elem), false, servicecommon.ResourceTypeNetworkPolicy)
+		err = r.Service.DeleteSecurityPolicy(types.UID(elem), true, servicecommon.ResourceTypeNetworkPolicy)
```

Comment:
This is because the previous DeleteSecurityPolicy is defined as:
`func (service *SecurityPolicyService) DeleteSecurityPolicy(obj interface{}, isVpcCleanup bool, createdFor string)`
The second arg is only to check if it's for cleanup, but the function definition is changed to 
`func (service *SecurityPolicyService) DeleteSecurityPolicy(obj interface{}, isVPCCleanupOrGC bool, createdFor string)`
The second arg is to check if its' for cleanup or GC.  
Here, the code is for GC process, so I changed it from `false` to `true`.

The reason why I changed `isVpcCleanup` to `isVPCCleanupOrGC`
is in https://github.com/vmware-tanzu/nsx-operator/pull/670/files#diff-53bc6d3b74f0d34b4187204c1bcf7ce4afe93c7f73e60b5de43c40dc44b88af0R721